Packs/FeedGCPWhitelist/Integrations/FeedGCPWhitelist/FeedGCPWhitelist.yml

```diff
@@ -0,0 +1,99 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: GCP Whitelist Feed
+  version: -1
+configuration:
+- defaultvalue: 'true'
+  display: Fetch indicators
+  name: feed
+  required: false
+  type: 8
+- additionalinfo: Indicators from this integration instance will be marked with this
+    reputation
+  defaultvalue: Good
+  display: Indicator Reputation
+  name: feedReputation
+  options:
+  - None
+  - Good
+  - Suspicious
+  - Bad
+  required: false
+  type: 18
+- additionalinfo: Reliability of the source providing the intelligence data
+  defaultvalue: A - Completely reliable
+  display: Source Reliability
+  name: feedReliability
+  options:
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  required: true
+  type: 15
+- display: ""
+  name: feedExpirationPolicy
+  defaultvalue: suddenDeath
+  type: 17
+  required: false
+  options:
+  - never
+  - interval
+  - indicatorType
+  - suddenDeath
+- defaultvalue: '20160'
+  display: ""
+  name: feedExpirationInterval
+  required: false
+  type: 1
+- defaultvalue: '30'
```

Comment:
Was there any information on the GCP page about how often the domains are typically updated? Cause it could be the case that fetching every 30 minutes is unnecessarily frequent for this feed integration. If you can't find anything about it, I'd suggest changing it to 4 or 6 hours.